scss/ionic.app.scss

```diff
@@ -47,6 +47,23 @@
 .extended-all {
   height: 101px;
 }
+#saved-notification {
```

Comment:
Should we be editing this file or instead making our own scss/css files for custom styles? @michaeljng